### ISSUE TITLE
Esc always removes search

### ISSUE THREAD

**Issue by folofjc** [Wojnach/vimflowy#23]
When I am in either insert or normal mode, `<esc>` always removes any search. Is this on purpose?

For instance, I am at a very top level. I search for something using `/`. I then navigate around to an item, and do something in insert mode. I then want to go back to normal mode and hit `<esc>`. It then removes the search that I had, which is not expected. The only way I know to then get out of insert mode is to do `<jk>`. However, my `vim` muscle memory always hits `<esc>` after typing, and removing the search is not expected.

**Comment by Wojnach** [comment]
Yeah this might have been my bad. Since jk is working then it should be possible to fix this for Esc aswell. Esc is natively bound to the workflowy search, so that might be why, but I'll have a look.

**Comment by folofjc** [comment]
Thanks so much, this would be a very helpful fix!
